os.bpmsuite.executionserver/added/launch/bpmsuite-executionserver.sh

```diff
@@ -160,6 +160,13 @@ function configure_EJB_Timer_datasource {
                     EJB_TIMER_XA_CONNECTION_PROPERTY_DatabaseName=${databaseName}
                     EJB_TIMER_XA_CONNECTION_PROPERTY_ServerName=${serverName}
                     EJB_TIMER_XA_CONNECTION_PROPERTY_PortNumber=${portNumber}
+
+                    if [ "${EJB_TIMER_DRIVER}" = "db2" ]; then
+                        # default to 4, but user could define 2
+                        local driverType=$(find_env "${dsPrefix}_DRIVER_TYPE" "4")
```

Comment:
AS the allowed values are only 2 or 4, shouldn't we add a verification to default it to 4 if the value provided is different than the allowed?

Reply:
@spolti I don't think it's necessary. We have so many places in our scripts where we rely upon the user specifying a correct value. For example, ports have to be numbers, and we don't check to make sure they don't contain letters.